pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>uk.gov.companieshouse</groupId>
         <artifactId>companies-house-parent</artifactId>
-        <version>1.2.0</version>
+        <version>1.3.0</version>
```

Comment:
Looks good .
There is repositories section which is not required. 
The jacoco version is also not needed as it is the same as in the parent pom.

Reply:
@divya051988 I have check removing the repositories section but then the install fails. Is there another step required here such as setting a proxy?